app/src/main/res/layout/post_list_item.xml

```diff
@@ -107,7 +107,7 @@
             android:layout_height="wrap_content"
             android:layout_marginStart="@dimen/small_16dp"
             android:layout_marginTop="@dimen/small_16dp"
-            android:text="@{instagramPost.likes}"
+            android:text="700"
```

Comment:
Instead of android:text="700", you can use **tools:text="700"** to check the ui in layout preview